polytracker/src/polytracker/main.cpp

```diff
@@ -280,8 +287,10 @@ void polytracker_start() {
   polytracker_print_settings();
   output_db = db_init(polytracker_db_name);
   // Store new file
-  input_id = storeNewInput(output_db, target_file, byte_start, byte_end,
-                           polytracker_trace);
+  for (auto target_source : target_sources) {
+    input_id = storeNewInput(output_db, target_source, byte_start, byte_end,
```

Comment:
What happens if `target_sources` is empty? Do we call `storeNewInput` if/when we encounter a new input?